news/2022-11-02-new-featured-artist-happy30.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "New Featured Artist: happy30"
+date: 2022-11-02 11:00:00 +0000
+---
+
+From composing circles in the editor to composing his own songs, **happy30** joins our Featured Artists!
+
+![](https://assets.ppy.sh/artists/317/header.jpg)
+
+You might not know **happy30** as a musician yet, but you're most definitely familiar with [**happy30** as a legendary osu! mapper](https://osu.ppy.sh/users/27767). 
+
+Whether it's fame from multi-million play beatmaps like [Demetori - Emotional Skyscraper ~ World's End](https://osu.ppy.sh/beatmapsets/13223) or infamy for [death](https://osu.ppy.sh/beatmapsets/42234) [spinners](https://osu.ppy.sh/beatmapsets/24611), **happy30** is bound to be somewhere in your songs library. He's been mapping since 2009, and somewhere between then and now, he's expanded his connection to music into song production!
+
+Try your hand at any of the **6** songs from [**happy30**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/317), including the full *As the wind blows* EP designed specifically for osu!.
+
+That's enough reading. You want to click things and hear noises. Start by clicking this big rectangle (also known as a video) showcasing osu!(lazer)'s new Freeze Frame mod on [one of **happy30**'s new tunes](https://osu.ppy.sh/beatmapsets/1873680), then click through the rest of the beatmaps and previews below:
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://assets.ppy.sh/artists/317/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### happy30 - You spin my world
+
+Check out [the map from the video above](https://osu.ppy.sh/beatmapsets/1873680) hosted by [Kyouren](https://osu.ppy.sh/users/2013571)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/317/As%20the%20wind%20blows%2Fhappy30%20-%20You%20spin%20my%20world.mp3" type="audio/mpeg">
+</audio>
+
+### happy30 - As the wind blows
+
+Or try playing [this beatmap](https://osu.ppy.sh/beatmapsets/355073) created by the musician himself!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/317/As%20the%20wind%20blows%2Fhappy30%20-%20As%20the%20wind%20blows.mp3" type="audio/mpeg">
+</audio>
+
+### happy30 - Relentless
```

Comment:
it might be worth mentioning maybe that happy30 himself has also mapped Relentless

http://osu.ppy.sh/beatmapsets/1177936#osu/2456625